crates/revm/revm-inspectors/src/stack/mod.rs

```diff
@@ -23,7 +23,7 @@ pub enum Hook {
     /// Hook on a specific block.
     Block(u64),
     /// Hook on a specific transaction hash.
-    Transaction(TxHash),
+    Transaction(B256),
```

Comment:
There's actually a `TxHash` alias in `alloy_primitives` which is equivalent to `B256`